docs/source/developer/intro.rst

```diff
@@ -4,12 +4,91 @@ Introduction
 .. contents:: Contents:
     :local:
 
-Purpose
-=======
 
 This guide provides information for those interested in participating in OpenDP development.
 
 If your focus is using OpenDP to build your own applications, please consult the :doc:`../user/index`.
 
+How to Get Involved
```

Comment:
I like the way you restructured this, with enumerating the ways to contribute, each with pointers how to get started.